lib/kitchen/directions/bake_numbered_exercise/main.rb

```diff
@@ -3,18 +3,14 @@
 module Kitchen
   module Directions
     module BakeNumberedExercise
-      def self.v1(exercise:, number:, suppress_solution: false, note_suppressed_solutions: false)
-        V1.new.bake(exercise: exercise, number: number, suppress_solution: suppress_solution,
+      def self.v1(exercise:, number:, suppress_solution_if: false, note_suppressed_solutions: false)
```

Comment:
@jpslav I have added a small fix above because of this error I've got while baking the recipe with changes from your PR: 
![image](https://user-images.githubusercontent.com/55203598/119657339-10862f80-be2c-11eb-8ae7-200ffb428f06.png)